Misc/NEWS.d/next/Library/2017-08-27-14-03-50.bpo-28638.W4CQxG.rst

```diff
@@ -0,0 +1,2 @@
+Speed up namedtuple class creation by 4x by avoiding usage of exec(). Patch
```

Comment:
This could be read as exec was eliminated, so perhaps '...by reducing usage of exec().'?